### DISCUSSION TITLE
[Feature proposal] Show more metadata on cards

### DISCUSSION THREAD

**Discussion by shamoon** [jonaswinkler/paperless-ng#439] [Archived FRs]
This has come up a couple times, Im not really wedded to this idea in general but there is something I like about both versions. Cards dont get any taller. Just throwing this out there 😬

Info icon, can put all in one tooltip:
<img width="546" alt="Screen Shot 2021-01-24 at 9 33 45 PM" src="https://user-images.githubusercontent.com/4887959/105665429-e7f52580-5e8b-11eb-978c-be3c9a1fd9fb.png">
<img width="523" alt="Screen Shot 2021-01-24 at 9 33 35 PM" src="https://user-images.githubusercontent.com/4887959/105665432-ea577f80-5e8b-11eb-81f5-627e168630c6.png">

Much busier but something I like about a bunch of little icons in a row:
<img width="1357" alt="Screen Shot 2021-01-24 at 9 46 56 PM" src="https://user-images.githubusercontent.com/4887959/105666247-b3826900-5e8d-11eb-9c74-410a3ac91410.png">
<img width="1100" alt="Screen Shot 2021-01-30 at 3 07 43 PM" src="https://user-images.githubusercontent.com/4887959/106370232-ed7bc100-630c-11eb-999b-570139500b76.png">

Again, just spitballing...

Edit: wrong images

**Comment by rknightion** [comment]
Personally, I like the second set. But that's potentially because I like to get as much information as possible (without being overwhelmed) as easily as possible. But I imagine that boils down to personal preference :).

**Comment by shamoon** [comment]
This will be solved by #748
